source/common/network/upstream_server_name.cc

```diff
@@ -0,0 +1,12 @@
+#include "common/network/upstream_server_name.h"
+
+namespace Envoy {
+namespace Network {
+
+absl::string_view UpstreamServerName::key() {
+  // Construct On First Use Idiom: https://isocpp.org/wiki/faq/ctors#static-init-order-on-first-use
+  static const char* cstring_key = "envoy.network.upstream_server_name";
```

Comment:
You can use the existing macro for this, https://github.com/envoyproxy/envoy/blob/15706964a29e595c045a5d7ef0646d04f347dcc1/source/common/common/version.cc#L13, here.

Reply:
Done in https://github.com/envoyproxy/envoy/pull/4973/commits/b31e6f66f1e9771c7525621f5e4923fa688b9a85